src/test/java/org/jabref/gui/externalfiles/AutoSetFileLinksUtilTest.java

```diff
@@ -36,10 +38,16 @@ public class AutoSetFileLinksUtilTest {
     private final BibDatabaseContext databaseContext = mock(BibDatabaseContext.class);
     private final BibEntry entry = new BibEntry(StandardEntryType.Article);
     private Path path = null;
+    private Path A;
+    private Path B;
 
     @BeforeEach
     public void setUp(@TempDir Path folder) throws Exception {
-        path = folder.resolve("CiteKey.pdf");
+        A = folder.resolve("A");
+        Files.createDirectory(A);
+        B = folder.resolve("B");
+        Files.createDirectory(B);
```

Comment:
Please do not modify *all* tests. Please move that initalization code to your tests (if possible)